.github/workflows/ci.yaml

```diff
@@ -5,8 +5,19 @@ on:
   - pull_request
 
 jobs:
-  ci:
+  pathogen-ci:
     uses: nextstrain/.github/.github/workflows/pathogen-repo-ci.yaml@master
     with:
       env: |
         NEXTSTRAIN_DOCKER_IMAGE: nextstrain/base
+
+  lint:
+     runs-on: ubuntu-latest
+     steps:
+     - uses: actions/checkout@v3
+     - uses: actions/setup-python@v4
+       with:
+         python-version: ~3
+     - run: pip install snakefmt
+     - run: snakefmt --check .
```

Comment:
I forgot to add in my original comment - along with the CI check, it would be equally appropriate to add developer documentation on how to check for and fix lint issues. Maybe create a `CONTRIBUTING.md` or `docs/DEV_DOCS.md` starting with this:

---

# Developer guide

## Snakemake formatting

We use [snakefmt](https://github.com/snakemake/snakefmt) to ensure consistency in style across Snakemake files in this project.

1. [Install snakefmt](https://github.com/snakemake/snakefmt#install)
2. Check for styling issues with `snakefmt --check .`
3. Automatically fix styling issues with `snakefmt .`

Reply:
Done! Also added `recommended workspace extensions` for VScode, making it easy to install snakefmt without searching for it.